lms/djangoapps/courseware/tests/mock_data.py

```diff
@@ -0,0 +1,86 @@
+
```

Comment:
I'm not sure this is the right place for this. Should this be under the openedx/core/djangoapps/program/tests directory?